frontend/src/Screen1.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react'
+import { navigate } from '@reach/router'
+import { css } from 'react-emotion'
+import { Trans } from '@lingui/macro'
+import { Form, Field } from 'react-final-form'
+import Button from '@govuk-react/button'
+import { H1, H3 } from '@govuk-react/header'
+import ListItem from '@govuk-react/list-item'
+
+const labelFormat = css`
+  margin-top: 20pt;
+`
+const submitButton = css`
+  margin-top: 20pt;
+`
+const listitem = css`
+  margin: 5pt;
+  margin-left: 20pt;
+`
+const textArea = css`
+  width: 500pt;
+  height: 200pt;
+`
+const onSubmit = () => {
+  navigate('/form2')
+}
+
+const validate = () => {}
+
+const MyForm = () => (
+  <Form
+    onSubmit={onSubmit}
+    validate={validate}
+    render={({ handleSubmit, pristine, invalid }) => (
+      <form onSubmit={handleSubmit}>
+        <H3 className={labelFormat}>
+          <label>
+            <Trans>What happened?</Trans>
+          </label>
+        </H3>
+        <div>
+          <Field
+            name="what happened"
+            component="textarea"
+            className={textArea}
+            placeholder=""
+          />
+        </div>
+
+        <Button
+          className={submitButton}
+          type="submit"
+          disabled={pristine || invalid}
+        >
+          <Trans>Submit</Trans>
+        </Button>
+      </form>
+    )}
+  />
+)
+
+export class Screen1 extends Component {
+  render() {
```

Comment:
Generally when you aren't using [React lifecycle methods](https://reactjs.org/docs/react-component.html#the-component-lifecycle) components should be written as [stateless functional components](https://hackernoon.com/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc). There are [varying levels of excitement about the "stateless functional"](https://twitter.com/jamiebuilds/status/1057313713835794432) parts but it's roundly considered a good idea. The recently announced [hooks](https://reactjs.org/docs/hooks-intro.html) are going to further reduce the appeal of class based components.

Reply:
this is good to know 👍